vector/src/main/java/im/vector/app/features/pin/lockscreen/pincode/PinCodeHelper.kt

```diff
@@ -36,16 +40,27 @@ class PinCodeHelper @Inject constructor(
      * Creates a PIN code key if needed and stores the PIN code encrypted with it.
      */
     suspend fun createPinCode(pinCode: String) {
-        val encryptedValue = lockScreenKeyRepository.encryptPinCode(pinCode)
+        val encryptedValue = lockScreenKeyRepository.encryptPinCode(pinCode.sha256())
         encryptedStorage.savePinCode(encryptedValue)
+
+        sharedPreferences.edit {
```

Comment:
I was thinking of adding the boolean to the actual storage (so in `SharedPrefPinCodeStore`), but I think this solution is fine.

Reply:
Done